LandSandBoat/server
# [core] Refactor to allow Dynamic Entities in Instances

**_I affirm:_**

## What does this pull request do?

All the regular foibles of entities in instances still apply. Having to look them up using the instance object, etc.

<img width="1427" alt="image" src="https://github.com/LandSandBoat/server/assets/1389729/12eaf0cf-8e1e-4a28-85f0-31cf14069d21">

## Steps to test these changes

- `!instance 7400`
- `!fafnir`
- Check to make sure `!fafnir` still works in regular zones, and the test DEs in GM Home are still there.

Closes https://github.com/LandSandBoat/server/issues/2030
